etc/org.opencastproject.distribution.aws.s3.AwsS3DistributionServiceImpl.cfg

```diff
@@ -24,3 +24,7 @@ org.opencastproject.distribution.aws.s3.distribution.enable=false
 # The AWS secret access key to use for connecting to S3.
 #org.opencastproject.distribution.aws.s3.secret.key=
 
+# Job loads
+job.load.aws.s3.distribute=1.0
```

Comment:
Can we put these in the same form I suggested for your other pull request` For example:

```
# Description of what job this load is for.
# Default: 1.0
#job.load.aws.s3.distribute=1.0
```